Hearthstone Deck Tracker/Importing/Websites/Marduktv.cs

```diff
@@ -0,0 +1,56 @@
+﻿#region
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Web;
+using Hearthstone_Deck_Tracker.Hearthstone;
+using Hearthstone_Deck_Tracker.Utility.Logging;
+
+#endregion
+
+namespace Hearthstone_Deck_Tracker.Importing.Websites
+{
+	public static class Marduktv
+	{
+		public static async Task<Deck> Import(string url)
+		{
+			return await ImportMarduktv(url);
+		}
+
+		private static async Task<Deck> ImportMarduktv(string url)
+		{
```

Comment:
There is no need to have this as a separate method, it can be directly in `Import`.